src/main/java/com/google/sps/servlets/FindLocationServlet.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/find-location")
+public class FindLocationServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    response.getWriter().println("<h1>Find Location</h1>");
```

Comment:
This is just temporary right?
Leave a TODO to fix this then.


Or better yet, set it up to return a 200 response instead of some dummy html (still include the todo to implement)

Reply:
Done